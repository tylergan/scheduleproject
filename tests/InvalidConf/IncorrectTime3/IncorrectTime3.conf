on Tuesday at 1260 run /bin/date