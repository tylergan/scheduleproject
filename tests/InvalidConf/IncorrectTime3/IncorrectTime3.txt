Testing that runner.py outputs the correct output to STDERR when there is:
    1) Incorrect time as the minutes do not range from [00, 60] but [00, 59] instead