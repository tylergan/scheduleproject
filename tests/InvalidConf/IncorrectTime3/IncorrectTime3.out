error in configuration: on Tuesday at 1260 run /bin/date
